timeless/poster/tasks.py

```diff
@@ -48,9 +48,10 @@ def sync_customers():
     Periodic task for fetching and saving tables from Poster
     Docs - https://dev.joinposter.com/docs/api#clients-getclients
     """
-    for poster_customer in __poster_api().customers():
-        customer = DB.session(Customer).query.filter_by(
-            poster_id=poster_customer["client_id"],
+    for poster_customer in __poster_api().customers().get('response', []):
+        customer = Customer.query.filter_by(
+            first_name=poster_customer["firstname"],
+            last_name=poster_customer["lastname"]
```

Comment:
@vladarefiev See comment from @emilianodellacasa [here](https://github.com/timelesslounge/timelessis/pull/522#discussion_r264058340). So, after all, the syncing should be done by poster_id <-> client_id, not first_name, last_name. 

I thought it was done in the PR where the discussion took place, but no, it's not done yet. Can you revert it back to your initial change? :D

Reply:
@amihaiemil done